extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -107,7 +109,17 @@ public static PhotoModel convertToPhotoModel(
         albumId.orElse(null),
         false  /*inTempStore*/,
         null  /*sha1*/,
-        mediaItem.getUploadedTime());
+        getUploadTime(mediaItem));
+  }
+
+  private static Date getUploadTime(GoogleMediaItem mediaItem)
+  {
+    Date uploadTime = null;
+    try {
+      uploadTime = FORMAT.parse(mediaItem.getMediaMetadata().getCreationTime());
```

Comment:
`mediaItem.getMediaMetadata().getCreationTime()`

if I understand how this will be constructed by jackson, this line will throw a null pointer exception if the JSON response from gphotos servers ever leave creation time field null, right?

Reply:
That is correct but according to the [documentation)](https://developers.google.com/photos/library/reference/rest/v1/mediaItems),  this field should never be null.